lsp-mode.el

```diff
@@ -5592,36 +5587,58 @@ SESSION is the active session."
       (lsp--start-workspace session client project-root (lsp--create-initialization-options session client))
     (lsp--spinner-stop)))
 
+;; lsp-log-io-mode
+
+(defvar lsp-log-io-mode-map
+  (let ((map (make-sparse-keymap)))
+    (define-key map (kbd "M-n") #'lsp--log-io-next)
+    (define-key map (kbd "M-p") #'lsp--log-io-prev)
+    (define-key map (kbd "k") #'lsp--erase-log-buffer)
+    map)
```

Comment:
The key bindings seem not conventional for Emacs.

Reply:
I don't use regular emacs bindings. What would you suggest ?

Reply:
I would suggest `M-k` to erase the whole buffer.
Or as @yyoncho's suggestion:

>Binding this to k feels a bit unnatural to me. What do you think about having k to delete only the current buffer logs and K to delete all?